### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
## Changes

- We don't need to have separate divs for every element.

Untitled design

- With div having the attribute `align="center"`, we don't need to provide any attributes to the children separately, it will inherit properties from its parent.

Untitled design (1)

- remove the extra line (No- 21)

### CODE REVIEW

File: profile/README.md
Comment (krishguptadev) [inline profile/README.md:1]:
```suggestion
<div align="center">
```

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by Pradumnasaraf** [comment]
Hey! @schmelto this PRis open from 9 days :)

**Comment by schmelto** [comment]
> Hey! @schmelto this PR is open from 9 days :)

Yeah normally in our maintainer guidelines two maintainers have to approve the PR before merging. But I think I will merge this PR anyway. Sorry for any inconveniences.

**Pull request merged**
